README.md

```diff
@@ -240,21 +240,145 @@ Similar to providers, plugins are independent projects that conform to the Dayto
+### Open Source Licensing
 
-1. Read the Developer Certificate of Origin Version 1.1 (https://developercertificate.org/)
-2. Signing all commits to the Daytona project.
+Daytona is Open Source under the [Apache License 2.0](LICENSE), and is the [copyright of its contributors](NOTICE). 
 
-This ensures that users, distributors, and other contributors can rely on all the software related to Daytona being contributed under the terms of the [License](LICENSE). No contributions will be accepted without following this process.
+If you would like to contribute to the software, you must:
 
-Afterwards, navigate to the [contributing guide](CONTRIBUTING.md) to get started.
+  1. **Read the Developer Certificate of Origin Version 1.1** 
 
-## License
+      Please review the [Developer Certificate of Origin Version 1.1](https://developercertificate.org/) to understand the contribution requirements.
+
+  2. **Sign all commits to the Daytona project** 
+
+      Ensure that all your commits are signed to comply with the Daytona project's contribution policies.
+
+      This ensures that users, distributors, and other contributors can rely on all the software related to Daytona being contributed under the terms of the [Apache License 2.0](LICENSE). No contributions will be accepted without following this process.
+
+### Ways to Contribute
+
+### 1. Reporting Issues and Suggesting Features
+
+Creating issues is a valuable way to contribute by reporting bugs, suggesting features, or improving documentation.
+
+Before creating a new issue, search the existing issues [here](https://github.com/daytonaio/daytona/issues) to see if your concern has already been addressed.
+
+* If no existing issue matches your contribution, follow these steps:
+   1. **Identify the Type of Issue**
+      * **Bug Report:** If you encounter unexpected behavior or errors.
+      * **Feature Request:** If you have an idea for a new feature or improvement.
+      * **Documentation Improvement:** If you notice gaps or areas for improvement in the documentation.
+   2. **Create a new issue**
+      * Navigate to Issues: Go to the Issues tab [here](https://github.com/daytonaio/daytona/issues).
+      * Click on "New Issue": Choose the appropriate template (Bug Report, Feature Request, etc.) if available.
+      * Fill Out the Issue Template: Provide a clear and concise description of the issue, including steps to reproduce (for bugs) or detailed feature descriptions.
+      * Submit the Issue: Click "Submit new issue" to create the issue.
+   3. **Engage with the Community**
+      * **Respond to Feedback:** Be prepared to provide additional information or clarification if maintainers or other contributors have questions.
+      * **Collaborate on Solutions:** If you have ideas for resolving the issue, share them in the comments.
+
+### 2. Contributing Code
+
+If you're interested in contributing code to Daytona, follow these steps:
+
+1. **Fork the Daytona repository**
```

Comment:
Before the "create" command I suggest adding a first step of checking if a GitHub provider is registered with `daytona git-provider add` so that pushing is possible. 
By the way, ordered lists using just 1., 1., 1. instead of 1, 2, 3 can be useful here https://www.markdownguide.org/basic-syntax/#ordered-lists

Reply:
made the changes